webapp/channels/src/components/sidebar_right/sidebar_right.tsx

```diff
@@ -155,7 +155,7 @@ export default class SidebarRight extends React.PureComponent<Props, State> {
 
         if (this.props.isOpen && (contentChanged || (!wasOpen && isOpen))) {
             this.previousActiveElement = document.activeElement as HTMLElement;
-            requestAnimationFrame(() => {
+            setTimeout(() => {
```

Comment:
using the requestAnimationFrame was causing the cypress headless tests to fail, using the setTimeout with a wait of 0 also does the trick and is more robust in test scenarios. I initially used the requestAnimationFrame for its and smooth UI updates and superior performance benchmark for animations and visible UI work, but in this case the minimum sacrifice in _**theoretical**_ performance optimization is outweighted for the increased reliability and stability of tests, especially when the UI is not be actively rendered like in cypress.